packages/lake/src/components/LakeTooltip.tsx

```diff
@@ -109,6 +109,7 @@ type Props = {
   width?: number;
   togglableOnFocus?: boolean;
   containerStyle?: ViewStyle;
+  showTooltip?: boolean;
```

Comment:
personal opinion: I don't like this name because it looks like a controlled component. I think just `disabled` is good for me. We can read it as "The tooltip is disabled so it will not be displayed on hover".  
What do you think?